x-pack/solutions/security/plugins/elastic_assistant/server/lib/prompt/tool_prompts.ts

```diff
@@ -151,4 +151,12 @@ If no relevant information is found, inform the user you could not locate the re
       default: 'Call this for Elastic Defend insights.',
     },
   },
+  {
+    promptId: 'IntegrationKnowledgeTool',
+    promptGroupId,
+    prompt: {
+      default:
+        'Call this for knowledge from Fleet-installed integrations, which contains information on how to configure and use integrations for data ingestion.',
+    },
+  },
```

Comment:
We've discussed this offline, but just commenting for visibility that we'll need to do release of the `security_ai_prompts` integration as the [bot mentioned](https://github.com/elastic/kibana/pull/236197#issuecomment-3348318257) once this is merged.

While technically I don't think you _need_ to do this (it'll just fall back to this local prompt defined here), it's a nice to have option for tweaking the prompt out of band. cc @stephmilovic

Reply:
yep! I have that as a to-do item in this PR description, instructions recommend to do the integration update after merge as well